src/Plotly.NET.Interactive.Extension/Extension.fs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Plotly.NET.Interactive.Extension
+
+open System.Threading.Tasks
+open Microsoft.DotNet.Interactive
+open Microsoft.DotNet.Interactive.Formatting
+open Plotly.NET.GenericChart
+
+type PlotlyNetFormatterKernelExtension() =
```

Comment:
I think this can also be named just `FormatterKernelExtension` as the full name is then`Plotly.NET.Interactive.FormatterKernelExtension`